src/test/scala/nl.knaw.dans.easy.validatebag/EasyValidateDansBagServletSpec.scala

```diff
@@ -29,6 +29,7 @@ import org.scalatra.test.scalatest.ScalatraSuite
 class EasyValidateDansBagServletSpec extends TestSupportFixture
   with EmbeddedJettyContainer
   with ScalatraSuite {
+  System.setProperty("http.agent", "Test")
```

Comment:
Can this be move to TestSupportFixture. Also, this would warrant a comment along the lines of "Some servers will deny requests if the User-Agent is set to the default value for Java".